sound.js

```diff
@@ -159,9 +166,17 @@ Sound.prototype.setNumberOfLoops = function(value) {
 
 Sound.prototype.setSpeed = function(value) {
   this._setSpeed = value;
```

Comment:
@sunjae1294 @ghsdh3409 Does anyone know what this variable is for?

Reply:
I'm not sure. I think it is some kind of a typo, but I just left it just in case it has some hidden usage.